.github/workflows/run_tests.yml

```diff
@@ -17,86 +17,33 @@ jobs:
        (github.event_name == 'pull_request' && !contains(github.event.pull_request.labels.*.name, 'no_ci'))
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest, macos-13]
+        config:
+        - os: ubuntu-latest
+          if: true
```

Comment:
Dislike the tag `if`. Doesn't tell me what it's meant to do.

Reply:
How about "run"? I guess I could use "skip" and invert all the logic, but it makes it a bit harder to see "ubuntu-latest, true, true, true" as the "do everything" case.

Reply:
`noskip: false` is a lot of negation to handle 😆

Reply:
Ok I am going with "skip"